devito/operator/operator.py

```diff
@@ -460,6 +460,12 @@ def _lower_iet(cls, uiet, profiler=None, **kwargs):
 
         # Lower IET to a target-specific IET
         graph = Graph(iet, **kwargs)
+
+        # Complex header if needed. Needs to be done before specialization
+        # as some specific cases require complex to be loaded first
```

Comment:
for instance?

Reply:
FFTW requires `complex.h` to be loaded first so that it's the type used rather than `fftw_complex`

Reply:
by "loaded first" you mean that the header file should stay at the very top of the includes list?

Reply:
yes

Reply:
It doesn't really matter right now but might later